app/templates/service.html

```diff
@@ -86,7 +86,7 @@ <h2 class="govuk-heading-m app-summary-list-heading" id="service-documents">Serv
       </li>
     {% endfor %}
     </ul>
-    <a class="govuk-link" href="#main-content" class="return-to-top">Return to top ↑</a>
+    <a class="govuk-link govuk-body" href="#main-content">Return to top ↑</a>
```

Comment:
I wonder if we can get rid of this link altogether...

Reply:
Considering we don't seem to use this anywhere else, and there's room for confusion with back to top links, I don't necessarily disagree...